src/rpc/misc.cpp

```diff
@@ -171,9 +188,13 @@ UniValue validateaddress(const UniValue& params, bool fHelp)
             "  \"isvalid\" : true|false,       (boolean) If the address is valid or not. If not, this is the only property returned.\n"
             "  \"address\" : \"navcoinaddress\", (string) The navcoin address validated\n"
             "  \"scriptPubKey\" : \"hex\",       (string) The hex encoded scriptPubKey generated by the address\n"
+            "  \"stakingaddress\" : \"navcoinaddress\", (string) The navcoin staking address part of a cold staking address\n"
+            "  \"spendingaddress\" : \"navcoinaddress\", (string) The navcoin spending address part of a cold staking address\n"
             "  \"ismine\" : true|false,        (boolean) If the address is yours or not\n"
+            "  \"ismine\" : true|false,        (boolean) If the coins from the address are stakable or not\n"
```

Comment:
Should be labelled as isstakable.

Reply:
already fixed in current master

Reply:
This pull request is already closed and merged. I would suggest to better check against the current state of master.

Reply:
Thought I did, but I was checking master on the source repo of the PR, not navcoin/navcoin-core. Will pay more attention in future.